### ISSUE TITLE
Keys on keyboard not working

### ISSUE THREAD

**Issue by elferherrera** [helix-editor/helix#2325]
It seems that on a non english keyboard certain keys are not identify with in normal mode. This is concerning because one cannot access the diagnostic menu that appears with ]. The same issue is found with |.

Recently this was fixed for the command mode. I hope something similar can be done for normal mode

**Comment by the-mikedavis** [comment]
If you're on unix can you see what the keycodes are by using `showkey -a`? I'm not sure if there's a windows equivalent. It might also be worth testing in another terminal emulator - it probably won't help but it's worth a shot to see if your current terminal emulator is swallowing the keypresses.

**Comment by elferherrera** [comment]
I don't think it is related to the terminal emulator. These keys can be used in insert, search and command (:) modes. They don't get registered in normal mode.

Does helix use a different event parsing for each of these cases?
